fathom/src/lang/core.rs

```diff
@@ -83,6 +83,8 @@ pub struct StructType {
     pub doc: Arc<[String]>,
     /// Name of this definition.
     pub name: String,
+    /// Parameter telescope.
```

Comment:
Ahh, I guess this isn't really a telescope just yet - the parameters can't depend on each other yet!